### ISSUE TITLE
vsphere post-processor should emit STDERR and STDOUT

### ISSUE THREAD

**Issue by hh** [mitchellh/packer#3808]
``` feature
Scenario: generate ovftool STDERR and STDOUT 

In order to debug ovftool errors
A packer user
wants to see verbose output from the ovftool command

Given I have the vsphere post-processor enabled
When and I run packer build -debug
Then I should see STDERR and STDOUT for any errors
```

**Comment by jmellor** [comment]
Problem confirmed.  I just lost multiple days of effort because of an inability to debug the vsphere post-processor for this reason.  I needed to accept an ssl cert, and stdout/stderr was never forwarded, resulting in a silent log and hung build consuming cpu forever.  This issue is critical to initial packer runs, as the debugging is totally insufficient to determine the cause of the hung build.

**Comment by jmellor** [comment]
`# Manual run of the internally-generated commandline to debug:
$ ovftool --acceptAllEulas --name=packer-ubuntu-1404.x64-2016.10.19.06.55.56 --datastore=Local_Datastore_50 --diskMode=thin output-qa.ubuntu.builder\\packer-qa.ubuntu.builder.vmx vi://qa_development:mypasswd@vcenter.internal/QA/host/QA%20Cluster
Opening VMX source: output-qa.ubuntu.builder\packer-qa.ubuntu.builder.vmx
Accept SSL fingerprint (xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx) for host vcenter.internal as target type.
Fingerprint will be added to the known host file
Write 'yes' or 'no'
yes
Opening VI target: vi://qa_development@vcenter.internal:443/QA/host/QA%20Cluster
Deploying to VI: vi://qa_development@vcenter.internal:443/QA/host/QA%20Cluster
Transfer Completed
Completed successfully`

This critical request/response is never passed to the user, causing a hung build.

**Comment by mwhooker** [comment]
I believe this has been solved in #3321
